### ISSUE TITLE
unable to use custom path for bin c:/opscode to c:/mypath/opscode

### ISSUE THREAD

**Issue by lbn-chef-team** [chef-cookbooks/chef_client_updater#68]
Hello,

### Cookbook version
3.0.4

### Chef-client version
chef 13.0.118 to 13.4.28

### Platform Details
Windows 2012 r2 x64

### Scenario:
update chef-client to the same custom path as defined in bootstrap installation

### Steps to Reproduce:
Install chef-client with 13.0.118 with path c:\mypath\opscode (instead of default c:\opscode)
Run the chef_client_updater to latest version.

### Expected Result:
exepect to have the new version installed to custom path.

### Actual Result:
Actually the new version don't care about initial install path and installing to c:\opscode

May we have an attribute to define the path. This attribute will add an argument to msiexec as INSTALLDIR ?

Best Regards :)

**Comment by lbn-chef-team** [comment]
Hello,

I check the code and i suggest to add this:

in providers/default.rb line 178 to 180:
```
def chef_install_dir
  node['chef_client_updater']['chef_install_dir']
end
```

a new attributes in attributes/default.rb line 37:
```
case node['platform_family']
when 'windows'
  node['chef_client_updater']['chef_install_dir'] = 'c:/opscode/chef'
else
  node['chef_client_updater']['chef_install_dir'] = '/opt/chef'
end
```

Regards,

**Comment by lamont-granquist** [comment]
fixed by #80 but i don't think the omnibus-chef package supports this behavior so it won't actually work until omnibus-chef ever supports it

**Comment by lbn-chef-team** [comment]
Hello,

Thanks for the fix but i still got an error.

```
STDERR: Could not find a part of the path 'C:\opscode\chef_upgrade.ps1'.Could not find a part of the path 'C:\opscode\chef_upgrade.ps1'.
```

some path are still hardcoded here in providers/default.rb line 262:
``` ruby
Kernel.spawn("c:/windows/system32/schtasks.exe /F /RU SYSTEM /create /sc once /ST \"#{upgrade_start_time}\" /tn Chef_upgrade /tr \"powershell.exe -ExecutionPolicy Bypass c:/opscode/chef_upgrade.ps1 > #{chef_upgrade_log}\"")
 ```
it should be:
``` ruby
Kernel.spawn("c:/windows/system32/schtasks.exe /F /RU SYSTEM /create /sc once /ST \"#{upgrade_start_time}\" /tn Chef_upgrade /tr \"powershell.exe -ExecutionPolicy Bypass #{chef_install_dir}/chef_upgrade.ps1 > #{chef_upgrade_log}\"")
 ```

and here in providers/default.rb line 389:
``` ruby
Remove-Item "c:/opscode/chef_upgrade.ps1"
 ```
it should be:
``` ruby
Remove-Item "#{chef_install_dir}/chef_upgrade.ps1"
 ```
